paper/paper.md

```diff
@@ -161,40 +161,54 @@ def hybrid_function(x):
 	return jnp.sin(circuit(x)) ** 2
 ```
 
+```pycon
+>>> hybrid_function(0.543)
+array(0.70807342)
+```
+
 We can also consider an example that includes a classical optimization loop, such as optimizing a
 quantum computer to find the ground state energy of a molecule:
 
 ```python
+import pennylane as qml
+from catalyst import grad, for_loop, qjit
 import jaxopt
-from catalyst import grad
+from jax import numpy as jnp
 
-molecule = qml.data.load("qchem", molname="H3+")[0]
-n_qubits = len(molecule.hamiltonian.wires)
+mol = qml.data.load("qchem", molname="H3+")[0]
+n_qubits = len(mol.hamiltonian.wires)
 
 dev = qml.device("lightning.qubit", wires=n_qubits)
 
 @qjit
 @qml.qnode(dev)
 def cost(params):
-    qml.BasisState(molecule.hf_state, wires=range(n_qubits))
+    qml.BasisState(jnp.array(mol.hf_state), wires=range(n_qubits))
```

Comment:
to avoid an autograph/jax dispatching error

Reply:
I wonder if there is anything we can do to help sort these problems for users. I've noticed that it's not necessarily clear that most data should be in jax arrays.

Reply:
I suspect it's likely a bug; that `mol.hf_state` is accidentally being created using `autograd.numpy` and not vanilla numpy. Vanilla numpy+JAX should not raise a warning